packages/ai/src/ui/process-ui-message-stream.ts

```diff
@@ -105,6 +105,7 @@ export function processUIMessageStream<MESSAGE_METADATA = unknown>({
               if (messageMetadataSchema != null) {
                 await validateTypes({
                   value: mergedMetadata,
+                  // @ts-ignore TODO: Couldn't figure this one out yet ~@gr2m
```

Comment:
@lgrammel if you like have a look at this type error. It's probably something obvious that I'll figure out quickly after some rest.